target/linux/ramips/dts/RB750Gr3.dts

```diff
@@ -0,0 +1,111 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+/ {
+	compatible = "mediatek,mt7621-eval-board", "mediatek,mt7621-soc";
+	model = "MikroTik RB750Gr3";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x10000000>;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+
+	gpio-leds {
```

Comment:
might be a good idea to add a note here that the power led is not gpio controllable.